code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -1053,8 +1063,9 @@
 				if (beaker)
 						return 1
 				else
+						if(!user.drop_item())
+								return 1

```

Comment:
The news is, that entire file is like that, so unless I change everything to a single tab indentation, I have to do double tab indentation.